distro.py

```diff
@@ -37,6 +37,40 @@
 import argparse
 import subprocess
 
+if False:
```

Comment:
what's the rationale for this condition? is that to work with Python 2, where no `typing.TYPE_CHECKING` is available? if so, a short comment would help in the long term for other readers.

amending `# pragma: nocover` to this line should calm down that automated commenter here.

Reply:
Thanks for the review @funkyfuture. I have applied your suggestions to the latest revision.